pages/cadastro/index.public.js

```diff
@@ -129,12 +128,16 @@ function SignUpForm() {
             contrast
             sx={{ px: 2, '&:focus-within': { backgroundColor: 'canvas.default' } }}
           />
-          {errorObject?.key === 'username' && (
-            <FormControl.Validation variant="error">{errorObject.message}</FormControl.Validation>
+          <FormControl.Caption>Este nome será exibido publicamente.</FormControl.Caption>
```

Comment:
Acredito que o correto aqui seja "esse" no lugar de "este", já que o campo citado está acima da frase

```suggestion
          <FormControl.Caption>Esse nome será exibido publicamente.</FormControl.Caption>
```